cdc/changefeed.go

```diff
@@ -481,7 +481,7 @@ func (c *changeFeed) calcResolvedTs(ctx context.Context) error {
 	minCheckpointTs := c.targetTs
 
 	if len(c.taskPositions) == 0 {
-		minCheckpointTs = c.status.CheckpointTs
+		minCheckpointTs = c.status.ResolvedTs
 	} else if len(c.taskPositions) < len(c.taskStatus) {
 		return nil
```

Comment:
Is this better:
- check if `len(c.taskPositions) < len(c.taskStatus)`, do nothing
- then check if `len(c.taskPositions) == 0`, update `minCheckpointTs` to `c.status.ResolvedTs`
If owner has dispatched a table but processor doesn't start replicating it, we should wait the processor is ready?

Reply:
> If owner has dispatched a table but processor doesn't start replicating it, owner should wait until the processor is ready?

Yes, the owner should, and we using table lock to make the owner wait until the processor is ready now